build.gradle

```diff
@@ -13,6 +13,10 @@ repositories {
 	mavenCentral()
 }
 
+springBoot {
+	mainClass = "com.roadtriprunner.RoadTripRunner.RoadTripRunnerApplication"
+}
```

Comment:
what is this?

Reply:
so super annoying... but my local is finding two "main" classes even though I deleted one... its stuck in git. So this is me manually telling my machine which main to run.